bench-vortex/benches/compress.rs

```diff
@@ -152,13 +154,13 @@ fn benchmark_compress<F, U>(
 {
     // if no logging is enabled, enable it
     if !LOG_INITIALIZED.swap(true, Ordering::SeqCst) {
```

Comment:
All of this file is gone in #2399. That output is a println since we parse it out right now for our benchmarking page